source/class/qx/ui/basic/Atom.js

```diff
@@ -276,25 +276,44 @@ qx.Class.define("qx.ui.basic.Atom", {
       this._handleIcon();
     },
 
-    // property apply
-    _applyGap(value, old) {
-      this._getLayout().setGap(value);
-    },
-
     // property apply
     _applyShow(value, old) {
       this._handleLabel();
       this._handleIcon();
     },
 
+    __safeSetPropertyOnLayout(value, propertyName) {
+      if (value === undefined) {
+        return;
+      }
+      const layout = this._getLayout();
+      const propertyGetter = `get${qx.lang.String.firstUp(propertyName)}`;
+      if (layout[propertyGetter]) {
+        layout[propertyGetter](value);
```

Comment:
Something seems off here. The original code was calling `layout.setGap(value)` or `layout.setCenter(value)`, etc. The new code is calling `layout.getGap(value)` or `layout.getCenter(value)`, etc., which ignores the parameter and doesn't actually do anything, if I'm reading this correctly...???

Reply:
Exactly. It is a mistake.

Reply:
Seems that this is not tested. Could you enhance the tests please?

Reply:
Sorry, that's a complete lack of focus on my part. I'll get that corrected and add some tests later today.